menpowidgets/base.py

```diff
@@ -793,6 +800,8 @@ def update_info(img, image_is_masked, group):
             text_per_line.append("> {} landmark points".format(
                 img.landmarks[group].lms.n_points))
             n_lines += 1
+        if custom_str is not None:
+            text_per_line.append('> custom message: {}'.format(custom_str))
```

Comment:
Don't print the `'custom message:'` string, especially if the custom function will return multiple lines. The user can define the custom function to return any string they want. However, do keep the `'> '` in front of each new line that is added in the info tab.